lib/src/default_extension_map.dart

```diff
@@ -799,6 +800,7 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'tfi': 'application/thraud+xml',
   'tfm': 'application/x-tex-tfm',
   'tga': 'image/x-tga',
+  'tgz': 'application/gzip',
```

Comment:
Then, `tgz` files are GZip files as well. Tar files has no standard MIME

Reply:
I'm unsure about this one. Wikipedia suggests `application/x-gtar`
https://en.wikipedia.org/wiki/List_of_archive_formats

My inclination is to not include it at all.